docs/getting-started-guides/kubeadm.md

```diff
@@ -213,199 +278,298 @@ kube-system   canal-node-77d0h                  2/3       CrashLoopBackOff    3
+kubeadm deb packages and binaries are built for amd64, arm and arm64, following
+the [multi-platform
+proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/multi-platform.md).
 
-deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out if there's interest).
+deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out
+if there's interest).
 
-Currently, only the pod network flannel is working on multiple architectures. You can install it this way:
+Currently, only the pod network flannel is working on multiple architectures.
+You can install it this way:
 
-    # export ARCH=amd64
-    # curl -sSL "https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml?raw=true" | sed "s/amd64/${ARCH}/g" | kubectl create -f -
+``` bash
+export ARCH=amd64
+curl -sSL "https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml?raw=true" | sed "s/amd64/${ARCH}/g" | kubectl create -f -
+```
 
-Replace `ARCH=amd64` with `ARCH=arm` or `ARCH=arm64` depending on the platform you're running on.
-Note that the Raspberry Pi 3 is in ARM 32-bit mode, so for RPi 3 you should set `ARCH` to `arm`, not `arm64`.
+Replace `ARCH=amd64` with `ARCH=arm` or `ARCH=arm64` depending on the platform
+you're running on. Note that the Raspberry Pi 3 is in ARM 32-bit mode, so for
+RPi 3 you should set `ARCH` to `arm`, not `arm64`.
 
 ## Cloudprovider integrations (experimental)
 
-Enabling specific cloud providers is a common request, this currently requires manual configuration and is therefore not yet supported. If you wish to do so, 
-edit the `kubeadm` dropin for the `kubelet` service (`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`) on all nodes, including the master. 
-If your cloud provider requires any extra packages installed on host, for example for volume mounting/unmounting, install those packages.
-
-Specify the `--cloud-provider` flag to kubelet and set it to the cloud of your choice. If your cloudprovider requires a configuration
-file, create the file `/etc/kubernetes/cloud-config` on every node. The exact format and content of that file depends on the requirements imposed by your cloud provider.
-If you use the `/etc/kubernetes/cloud-config` file, you must append it to the `kubelet` arguments as follows:
-`--cloud-config=/etc/kubernetes/cloud-config`
-
-Lastly, run `kubeadm init --cloud-provider=xxx` to bootstrap your cluster with cloud provider features.
-
-This workflow is not yet fully supported, however we hope to make it extremely easy to spin up clusters with cloud providers in the future. 
-(See [this proposal](https://github.com/kubernetes/community/pull/128) for more information) The [Kubelet Dynamic Settings](https://github.com/kubernetes/kubernetes/pull/29459) feature may also help to fully automate this process in the future.
+Enabling specific cloud providers is a common request, this currently requires
```

Comment:
Replace comma with period. Second sentence becomes: "However, specific cloud providers require manual configuration and are therefore not supported by `kubeadm` in version 1.6."